www/doctl/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem			1.0
+PortGroup			golang 1.0
+
+go.setup			github.com/digitalocean/doctl 1.18.0 v
+revision			0
+
+checksums			rmd160	7d454c8812222227a28f8e4b1c27c9b37d9ef589 \
+					sha256	5f681494c3a9750e0cf944a140da2e23c117c8b8b06e474c96c73552a7162e5c \
+					size	15651051
+
+categories			www devel
+platforms			darwin
+license				Apache-2
+maintainers			{@kritr gmail.com:krdevmail} openmaintainer
+description			A command line interface for the DigitalOcean API
+long_description	${description}
```

Comment:
Minor comment, but the indentation above is a little wonky. Could you align things.